app/channels/application_cable/connection.rb

```diff
@@ -1,20 +1,20 @@
 module ApplicationCable
   class Connection < ActionCable::Connection::Base
-    attr_accessor :watched_link, :current_user
+    attr_accessor :watched_links, :current_user
 
     def connect
-      @watched_link = nil
+      @watched_links = []
```

Comment:
Ruby developer mindset, pluralizing like DHH having a dissociative psychotic event ❤️

Reply:
I'm just impressed I could push a Link object into it and everything just works™️

You know the kind of memory implications that would have in Rust??